app/controllers/api/v1/jobs/update.rb

```diff
@@ -0,0 +1,21 @@
+module API
+  module V1
+    module Jobs
+      class Update < Base
+        helpers Params
+
+        params do
+          requires :id, type: Integer, desc: 'Id of a certain job'
```

Comment:
Could you please explain why do you use `helpers Params` and `params` block here